src/styles/components/_buttons.scss

```diff
@@ -1,4 +1,4 @@
-@import 'utils/mixins';
+@import './../utils/mixins';
```

Comment:
I don't think you need this import at all since `_buttons` is imported after `_mixins` in main.scss.